backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseParquetSuite.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.glutenproject.execution
+
+import io.glutenproject.GlutenConfig
+import io.glutenproject.utils.UTSystemParameters
+
+import org.apache.spark.SparkConf
+import org.apache.spark.internal.Logging
+
+class GlutenClickHouseParquetSuite extends WholeStageTransformerSuite with Logging {
+  override protected val backend: String = "ch"
+  override protected val resourcePath: String = ""
+  override protected val fileFormat: String = "parquet"
+
+  protected val rootPath: String = getClass.getResource("/").getPath
+  protected val basePath: String = rootPath + "unit-tests-working-home"
+
+  protected val warehouse: String = basePath + "/spark-warehouse"
+
+  override protected def sparkConf: SparkConf = {
+    super.sparkConf
+      .set(GlutenConfig.GLUTEN_LIB_PATH, UTSystemParameters.getClickHouseLibPath())
+      .set("spark.gluten.sql.enable.native.validation", "false")
+      .set("spark.sql.warehouse.dir", warehouse)
+  }
+
+  test("native scan disabled case") {
+    withSQLConf(("spark.gluten.enabled", "false")) {
```

Comment:
why is disable gluten ?

Reply:
Use sparksql to create data first, then run actual test bellow.Just to make sure no error will occur.